treeherder/webapp/api/performance_data.py

```diff
@@ -492,3 +495,35 @@ def list(self, request):
 
         serializer = self.get_serializer(self.queryset, many=True)
         return Response(data=serializer.data)
+
+
+class ValidityDashboardViewSet(viewsets.ViewSet):
```

Comment:
I think this should be named based on the data it's providing, not what it will be used for (I know that's not the easiest thing but that's how we've generally named our endpoints).